source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRPresentationViewController.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ACRPresentationViewController.m
+//  ADCIOS
+//
+//  Created by jwoo on 5/31/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ACRPresentationViewController.h"
+#import "ACRViewController.h"
+
+@implementation ACRPresentationViewController
+
+- (id) init:(NSString*) str{
```

Comment:
>init:(NSString*) str [](start = 7, length = 20)

Can this view controller be created via storyboard, or only programmatically? If it's allowed in a storyboard (the common way of creating view controllers), this initializer will never be called.

More generally, the designated initializer for UIViewController is initWithNibName:bundle:, so you should be sure to override that method and make sure that all initializers in this class call that method on the superclass.